Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1266,7 +1266,7 @@
   icon: { sprite: /Textures/Objects/Fun/pai.rsi, state: syndicate-icon-pai-off }
   productEntity: SyndicatePersonalAI
   cost:
-    Telecrystal: 1
+    Telecrystal: 0 # DeltaV - was 1
```

Comment:
valid pai can use syndie radio so this is a no brainer for free, make this a discount instead